### PR TITLE
move auth to adminstration

### PR DESCRIPTION
**Pull request by manmeetkaur**
Related to the discussion about moving Auth items to Admin menu. This is a temp change. But rather than having too many changes in our docs and confuse users, I am inclined to first have a discussion on this https://github.com/astronomer/docs/issues/2787 and then do the shuffle. it not a big change, we just have to agree on the pattern.

### CODE REVIEW

File: astro/configure-deployment-resources.md [hunk 1]
```diff
@@ -171,22 +171,6 @@ To give your tasks more or less resources, change the worker type in the task's
 
 :::
 
-## Transfer a Deployment to another Workspace
```
Comment (jwitz) [inline astro/configure-deployment-resources.md:174]: We need a redirect for this

File: astro/transfer-a-deployment.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+sidebar_label: 'Transfer a Deployment'
+title: 'Transfer a Deployment to another Workspace'
+id: transfer-a-deployment
+description: "Transfer a Deployment to another Workspace in the same cluster"
```
Comment (jwitz) [inline astro/transfer-a-deployment.md:5]:
```suggestion
description: "Transfer a Deployment to another Workspace in the same cluster."
```

File: sidebarsAstro.js [hunk 3]
```diff
@@ -93,8 +93,6 @@ module.exports = {
         "api-keys",
         "environment-variables",
         "secrets-backend",
```
[the remark this answers is not in the document]
Comment (manmeetkaur) [reply sidebarsAstro.js:95]: manage DAGs is actually not something that you do in a Deployment and more of a Observability feature. So I think it better fits there. nvm. I reverted.

File: sidebarsAstro.js [hunk 4]
```diff
@@ -204,6 +201,13 @@ module.exports = {
             },
           ],
         },
+        {
+          type: "category",
+          label: "Organizations",
```
Comment (jwitz) [inline sidebarsAstro.js:206]:
```suggestion
          label: "Organization",
```
I'm fine either way but I feel like the vast vast majority of users will only ever have one Org, so it's okay to refer to it like that

**Review by jwitz — APPROVED** [review]
LGTM!

**Pull request merged**